server/paymentProviders/utils.js

```diff
@@ -174,7 +174,10 @@ export const persistTransaction = async (virtualCard, transaction) => {
       });
       if (originalCreditTransaction?.amount === amount) {
         if (!originalCreditTransaction.RefundTransactionId) {
-          await createRefundTransaction(originalCreditTransaction, 0, { refundTransactionId: transactionId });
+          await createRefundTransaction(originalCreditTransaction, 0, {
+            refundTransactionId: transactionId,
+            transaction,
+          });
```

Comment:
Logging the transaction here to later help resolve https://github.com/opencollective/opencollective/issues/7463